rcl/src/rcl/subscription.c

```diff
@@ -432,9 +433,19 @@ rcl_subscription_get_actual_qos(const rcl_subscription_t * subscription)
 bool
 rcl_subscription_can_loan_messages(const rcl_subscription_t * subscription)
 {
+  const char * env_val = NULL;
+
   if (!rcl_subscription_is_valid(subscription)) {
     return false;  // error message already set
   }
+  const char * env_error_str = rcutils_get_env(RCL_DISABLE_LOAN_MSG_ENV_VAR, &env_val);
+  if (NULL != env_error_str) {
+    RCUTILS_SAFE_FWRITE_TO_STDERR("Error getting env var: ");
+    RCUTILS_SAFE_FWRITE_TO_STDERR(RCL_DISABLE_LOAN_MSG_ENV_VAR);
+    RCUTILS_SAFE_FWRITE_TO_STDERR("\n");
```

Comment:
When the error string is not NULL, it should be included in the stderr outpput, I think. It returns more information, e.g.: https://github.com/ros2/rcutils/blob/1570091b559fe605fee5cfdf4a6a97f412836656/src/env.c#L71

Reply:
right, fixed.